tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_all_opcodes_in_container.py

```diff
@@ -337,19 +338,19 @@ def test_all_opcodes_stack_overflow(
     opcode = opcode[0] if opcode.has_data_portion() else opcode
 
     assert opcode.pushed_stack_items - opcode.popped_stack_items == 1
-    opcode_count = MAX_OPERAND_STACK_HEIGHT + 1 - opcode.min_stack_height
+    opcode_count = MAX_RUNTIME_STACK_HEIGHT - opcode.min_stack_height
 
     bytecode = Op.PUSH0 * opcode.min_stack_height
     bytecode += opcode * opcode_count
     bytecode += Op.STOP
 
-    kwargs: Dict[str, Any] = {"code": bytecode}
+    code_section = Section.Code(code=bytecode)
 
     if exception == EOFException.INVALID_MAX_STACK_INCREASE:
         # Lie about the max stack height to make the code be checked for stack overflow.
-        kwargs["max_stack_height"] = MAX_OPERAND_STACK_HEIGHT
+        code_section.max_stack_height = MAX_STACK_INCREASE_LIMIT
```

Comment:
This line is making the test filling fail due to assertion in https://github.com/ethereum/execution-spec-tests/blob/91687b77e66fe5bf5e4ea3612a68fc7876650bf2/src/ethereum_test_types/eof/v1/__init__.py#L208

If we rollback to using kwargs so that `max_stack_height` is set but `max_stack_increase` is not, then the test fails due to an incorrect expected exception:
```
raise EOFExceptionMismatchError(
E   ethereum_test_specs.eof.EOFExceptionMismatchError: EOF code raised a different exception than expected:
E       Code: ef00010100040200010401ff00000000800400303030303030...30303030303030303030303030303030303030303030303000
E   Expected: EOFException.INVALID_MAX_STACK_INCREASE
E        Got: EOFException.MAX_STACK_INCREASE_ABOVE_LIMIT (err: max_stack_increase_above_limit)
```